src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -1778,6 +1779,30 @@ class DistributedClosestPoint
     return success;
   }
 
+  void verifyTopologyName(const conduit::Node& meshNode,
+                          const std::string& topologyName)
+  {
+    std::string coordsetPath;
+    const std::string topologyPath =
+      axom::fmt::format("topologies/{}", topologyName);
+    for(axom::IndexType d = 0; d < meshNode.number_of_children(); ++d)
+    {
+      const auto& domain = meshNode.child(d);
+      if(!domain.has_path(topologyPath))
+      {
+        auto errMsg = fmt::format("No such topology '{}' found.", topologyName);
+        if(domain.has_path("coordsets/" + topologyName))
+        {
+          errMsg += fmt::format(
+            "  You may have mistakenly specified a coordset name."
+            "  The interface has changed to use topology name"
+            " instead of coordset.");
+        }
+        SLIC_ERROR(errMsg);
```

Comment:
Would it be helpful to list the domain ID or path to the `meshNode` to give users a better sense of where things went wrong?

Reply:
I don't think it's that helpful in practice, because the domains very typically have the same topology and coordset, even though blueprint allows for differences.